agent-zero/skills/blender-3d-studio/scripts/generate-animation.sh

```
@@ -0,0 +1,594 @@
+done
+
+# Validate required arguments
+if [ -z "$INPUT_FILE" ]; then
+    log_error "Input file is required (-i/--input)"
+    show_help
+    exit 1
+fi
+
+if [ -z "$OUTPUT_FILE" ]; then
+    # Generate default output filename
+    local base_name=$(basename "$INPUT_FILE")
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)